Misc/NEWS.d/next/Library/2020-05-03-12-55-55.bpo-40447.oKR0Lj.rst

```diff
@@ -0,0 +1 @@
+Accept pathlib.Path in compileall.compile_file.
```

Comment:
`compileall.compile_file` already accepts these for the *fullname* parameter; it's the other path parameters that this PR changes. This needs clarification.